activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -659,6 +659,10 @@ def initialize_type_map(m) # :nodoc:
         end
       end
 
+      def clear_case_insensitive_cache
+        @case_insensitive_cache = { "citext" => false }
```

Comment:
This is exposing code that is only needed for tests. We can set the instance variable in the test to `nil` instead

Reply:
Oh yes, that worked. Thank you!